Density/app/src/main/res/layout/facilities_activity.xml

```diff
@@ -99,6 +99,61 @@
                 android:layout_height="match_parent"
                 android:orientation="vertical">
 
+                <androidx.constraintlayout.widget.ConstraintLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:paddingTop="15dp"
+                    android:background="?android:attr/selectableItemBackground">
```

Comment:
ConstraintLayout should go inside the CardView so that it can constrain and set the positions of the images and text